contents/blog/what-is-a-growth-engineer.md

```diff
@@ -0,0 +1,134 @@
+---
+title: What is a growth engineer (and how they help your startup)
+date: 2023-04-20
+author:
+    - ian-vanagas
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+category: Engineering
+tags:
+ - Explainers
+---
+
+Every startup loves to see its growth metrics like signups, subscriptions, and revenue go up, yet nearly all engineers work on things indirectly related to doing so. Growth engineers are different. They focus entirely on projects dedicated to driving as much growth as possible, no matter where they build those projects.
+
+The post goes over growth engineers, with a specific focus on our technical version role at PostHog. We cover the type of people who succeed as growth engineers, their skills, what they work on, and why you should hire them.
```

Comment:
```suggestion
In this post, we'll explain the benefits of growth engineers, the type of people who succeed as growth engineers, their skills, what they work on, why you should hire them, and how growth engineering works at PostHog.
```

Reply:
Suggestion: I think we should add a short boxout to the end of this intro that's basically "Who are we?". Would add some useful context about our perspective + a chance to share our values with the reader.

Reply:
Got it